### PR TITLE
Wrap in http server

### PR DESCRIPTION
**Pull request by dirrelito**
I have with this fix wrapped the whole code in a http server. It accepts the two endpoints discussed in #2  with one exception: it accepts multiple words not as comma separated, but rather via multiple arguments:

    /check?words=hej&words=hopp

This is simply becuase thats the way Node native `http`-library works.

I have also added the nodemon dependancy to allow for `npm start` which make the webserver restart every time you edit the code. That is super good for dev purposes.

### CODE REVIEW

**Comment by dirrelito** [comment]
would be happy if @marcusahl could join this repo as contributor so i can ask for review!

**Comment by dirrelito** [comment]
I have prepared for future http loading of networks and paramers by making the loads asynchronously. This PR has been "completed" now.

**Comment by dirrelito** [comment]
a squash rebase will probably be nicest, so we dont keep this mass of commits

File: app.js [hunk 1]
```diff
@@ -1,39 +1,26 @@
...
+//// Routing
+app.get('/check',my_check.check);
+app.get('/reload',my_reload.reload);
+app.get('*',four_o_four); // the 404 route. captures all requests not captured above.
 
-console.log(zip(z_raw,z_hat))
\ No newline at end of file
+//// Fire up the show!
+my_reload.base_load(app)
+app.listen(port);
```
Comment (marcusahl) [inline app.js]: Whiteline

File: routes/reload.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	const prom = ml_util_promise();
+	prom.then(x => {
+		express_app.set("ml-util", x);
+	});	
+};
+
+module.exports = {
+	reload: reload,
+	base_load: base_load
+}
```
Comment (marcusahl) [inline routes/reload.js]: \n

File: routes/check.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+	} else {
+		res.status(422);
+		res.json({ err: "This endpoints need a GET parameter called 'words' which may be repeated, e.g. '/check?words=hej&words=hopp'"});
+	}
+}
+
+module.exports =  {
+	check: check
+}
```
Comment (marcusahl) [inline routes/check.js]: \n

File: app.js (hunk 1 above)
Comment (dirrelito) [inline app.js]: fixed!

File: routes/check.js (hunk 3 above)
Comment (dirrelito) [inline routes/check.js]: fixed

File: routes/reload.js (hunk 2 above)
Comment (dirrelito) [inline routes/reload.js]: fixed

**Comment by dirrelito** [comment]
I fixed the concrete points about three line endings.

Regarding the structure of the service, I try to keep stuff as simple as possible. As few lines of codes as possible this far. Up to this point, there have been no need to introduce complicated layering etc. 

What are you missing for accepting the PR?

**Comment by dirrelito** [comment]
Since I was in the mood for coding. I refactored a bit to separate web logic from other logic. I have also added some options-parameters for preparing the move to using a http data source. See issue #4 .

**Pull request merged**